pkg/sdk/logging/api/v1beta1/logging_types.go

```diff
@@ -258,6 +258,18 @@ func (l *Logging) SetDefaults() error {
 		if l.Spec.FluentdSpec.BufferVolumeImage.PullPolicy == "" {
 			l.Spec.FluentdSpec.BufferVolumeImage.PullPolicy = "IfNotPresent"
 		}
+		if l.Spec.FluentdSpec.BufferVolumeResources.Limits == nil {
+			l.Spec.FluentdSpec.BufferVolumeResources.Limits = v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("10M"),
+				v1.ResourceCPU:    resource.MustParse("1m"),
+			}
+		}
+		if l.Spec.FluentdSpec.BufferVolumeResources.Requests == nil {
+			l.Spec.FluentdSpec.BufferVolumeResources.Requests = v1.ResourceList{
+				v1.ResourceMemory: resource.MustParse("5M"),
+				v1.ResourceCPU:    resource.MustParse("1m"),
+			}
+		}
```

Comment:
```suggestion
		if l.Spec.FluentdSpec.BufferVolumeResources.Limits == nil {
			l.Spec.FluentdSpec.BufferVolumeResources.Limits = v1.ResourceList{
				v1.ResourceMemory: resource.MustParse("10M"),
				v1.ResourceCPU:    resource.MustParse("50m"),
			}
		}
		if l.Spec.FluentdSpec.BufferVolumeResources.Requests == nil {
			l.Spec.FluentdSpec.BufferVolumeResources.Requests = v1.ResourceList{
				v1.ResourceMemory: resource.MustParse("10M"),
				v1.ResourceCPU:    resource.MustParse("1m"),
			}
		}
```

Reply:
done in b68e60adea95bb053946c011f411205cd935b10f